src/components/CrazyInput/CrazyInput.vue

```diff
@@ -167,16 +167,26 @@
         >
           <!-- Chip selection -->
           <PawChip
-            v-for="item in chips"
+            v-for="item in maxChips>0 ? chips.slice(0 ,maxChips) : chips"
             :key="item.id"
             @clicked="removeChip(item)"
             size="sm"
             :icon="chipIcon"
             :title="this.t('removeItem')"
+            :textEllipsis="chipsTruncated"
             class="shrink align-top justify-self-start ml-1 mt-1"
             >{{ item.value }}</PawChip
           >
 
+           <!-- Chip elipsis item -->
+           <PawChip
+            v-if="maxChips> 0 && chips.length > maxChips"
```

Comment:
this condition should also be updated, maxChips && chips.length > maxChips should be enough